openstudiocore/src/utilities/geometry/Geometry.hpp

```diff
@@ -82,14 +83,14 @@ namespace openstudio{
 
   /// compute triangulation of vertices, holes are removed in the triangulation
   /// requires that vertices and holes are in clockwise order on the z = 0 plane (i.e. in face coordinates but reversed) 
-  UTILITIES_API std::vector<std::vector<Point3d> > computeTriangulation(const Point3dVector& vertices, const std::vector<std::vector<Point3d> >& holes, double tol = 0.001);
+  UTILITIES_API std::vector<std::vector<Point3d> > computeTriangulation(const std::vector<Point3d>& vertices, const std::vector<std::vector<Point3d> >& holes, double tol = 0.001);
 
   /// move all vertices towards point by distance, pass negative distance to move away from point
   /// no guarantee that resulting polygon will be valid
```

Comment:
This is a weird method. I'm not sure what your use case is, but it seems like you could make some strange shapes with this.